frontend/src/components/MovieList.jsx

```diff
@@ -47,33 +58,43 @@ function MovieList(props) {
     navigate(`../${value.alt}`);
   };
 
+  // console.log(props.id)
+
   return (
     <div>
-      <h1>Suggested Movies</h1>
-      <List>
-        {options.map((value) => (
-          <ListItem key={`${value.id}`}>
-            <StyledPaper elevation={3} className="Movie-body">
-              <span className="Movie-poster-review-panel">
-                <Paper elevation={3} className="Movie-poster">
-                  <img
-                    loading="lazy"
-                    width="250"
-                    src={`https://image.tmdb.org/t/p/original/${value.poster_path}`}
-                    className="Movie-poster"
-                    alt={value.id}
-                    onClick={handleClick}
-                  />
-                </Paper>
-                <span className="Movie-body">
-                  <h2>{value.title}</h2>
-                  <p>{value.overview}</p>
-                </span>
-              </span>
-            </StyledPaper>
-          </ListItem>
-        ))}
-      </List>
+      {props.id ? <h1> RECOMMENDED MOVIES</h1> : <h1>Suggested Movies</h1>}
```

Comment:
Here it is only rendering RECOMMENDED since we are always giving in id. We can change it to check if id is equal to -1.

Reply:
Oh, good catch!